CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # @digitalbazaar/ecdsa-multikey ChangeLog
 
-## 1.1.0 - 2023-XX-XX
+## 1.2.0 - 2023-XX-XX
+
+### Changed
+- Change exported algorithm from "ECDSA" to curve name.
+
+## 1.1.0 - 2023-03-02
 
 ### Changed
 - Ensure public and secret multikey headers match.
```

Comment:
It looks like v1.1.0 has not been released yet, so you can move your new changelog entry under the old one. Usually the publisher updates the date when making the release, so you can also just leave the date as TBD.

```suggestion
## 1.1.0 - 2023-XX-XX

### Changed
- Ensure public and secret multikey headers match.
- Change exported algorithm from "ECDSA" to curve name.
```

Reply:
Good to know 👍 Thanks Tashi.